dace/symbolic.py

```diff
@@ -855,12 +896,28 @@ def _print_Symbol(self, expr):
     def _print_Pow(self, expr):
         base = self._print(expr.args[0])
         exponent = self._print(expr.args[1])
+
+        # Special case for square root
+        try:
+            if float(exponent) == 0.5:
+                return f'dace::math::sqrt({base})'
+        except ValueError:
+            pass
+
+        # Special case for integer powers
         try:
             int_exp = int(exponent)
-            assert (int_exp > 0)
+            if int_exp == 0:
```

Comment:
Shouldn't this use the new `ipow` runtime method?

Reply:
Fixed

Reply:
I have to revert this. Intel FPGA with OpenCL doesn't like namespaces :(